chat-commands.js

```diff
@@ -2199,12 +2195,12 @@ exports.commands = {
 		let name = targetUser ? targetUser.name : this.targetUsername;
 
 		if (!userid) return this.parse('/help trustuser');
-		if (!targetUser) return this.errorReply("User '" + name + "' is not online.");
+		if (!targetUser) if (!targetUser) return this.errorReply(`User '${name}' is not online.`);
```

Comment:
This is clearly a typo.

Reply:
Honestly I don't even understand how I did this >_>